lib/shenzhen/commands/build.rb

```diff
@@ -32,6 +33,7 @@
     @xcconfig = options.xcconfig
     @xcargs = options.xcargs
     @destination = options.destination || Dir.pwd
+    @ipa_name_override = options.ipa || nil
```

Comment:
As far as I know, `options.ipa` will be nil by default if it's not given by the user